Cargo.lock

```diff
@@ -1257,7 +1257,7 @@ dependencies = [
  "libc",
  "option-ext",
  "redox_users",
- "windows-sys 0.61.0",
+ "windows-sys 0.59.0",
```

Comment:
Tweaked this to go up to 0.61.1 instead of down to 0.59.0 (like in other places).